### DISCUSSION TITLE
Ome Player test page show lagged stream

### DISCUSSION THREAD

**Discussion by antonymarion** [OvenMediaLabs/OvenMediaEngine#1685] [Help]
Hi all,

One of the end user showed me a lagged video stream while reading/using the test player of ome (deployed using the docker image) with WebRTC.

Nevertheless I could not see in my case any lag or bad quality video. it was working well for me using the same test player and the same stream input.

On top of that, and that is very strange to me, the same end user did not have any issue opening a new tab and selecting (input source field of the demo player) another stream source.

How could this happen and what is the solution, since I believe it comes from the player only and it is not due to network/infra issue otherwise it would be either global for all or global for the end users' streams showed inside the test player tabs.

N.B.: the same behaviour happened after the end user restarted its computer and only opened 2 tabs: the same stream that before was still  lagged on his computer (but was still working well for me on my tab), another stream was working well (no lag) in his computer (so not a network issue)

**Comment by nums** [comment]
If the stream lags, it is a network problem, wifi or even vpn / firewall of the company of your end user.

**Reply by antonymarion** [reply]
if it was this kind of pb, the end user would have lagged streams for both of the streams and that is not the case.

The end user had a stream A lagged, but a stream B OK.
On my PC the stream A is OK and the stream B is OK.

**Reply by getroot** [reply]
Aren't the bitrates of Stream A high?

**Reply by antonymarion** [reply]
Where can I have access to bitrate (on OME logs ? or can I see them on the player ?)

**Reply by antonymarion** [reply]
But anyway I suppose that bitrate was high; otherwise I could not be able at that time to see a smooth stream on my PC

**Reply by getroot** [reply]
Here's my scenario:
- Stream A had a high bitrate, and Stream B had a low bitrate.
- Your end user didn't have enough network bandwidth to watch A smoothly, but they had enough network bandwidth to watch B.

You can check the bitrate in your encoder, OME log, OME API, etc. If it's WebRTC, you can also check it in chrome://webrtc-internals.

**Reply by antonymarion** [reply]
Thks a lot, I will confirm this, but it seems a good explanation thks!

**Reply by antonymarion** [reply]
The bit rate of the 300 RTK DJI drone is the following: 

![image](https://github.com/user-attachments/assets/6f75cace-8f78-41a5-921e-4a36bb991841)

cf https://github.com/dji-sdk/Mobile-SDK-Android-V5/issues/341

So the max possible is 380kbyte/s which is not that big and should be pretty smooth in any type of connection right?

**Reply by getroot** [reply]
380kbyte/s is 3.04Mbps.

**Reply by antonymarion** [reply]
![image](https://github.com/user-attachments/assets/dcddd109-8323-4384-851c-5fda1601f0c4)

The connection was good at that time (we already checked it).

But both on our app and the ome test player (on the PC that shows the speed test) the stream A was lagging
